docs/core/docker/building-net-docker-images.md

```diff
@@ -2,224 +2,255 @@
+* `microsoft/dotnet:<version>-runtime-deps`: The **runtime-deps** image contains the operating system with all of the native dependencies needed by .NET Core. This image is for [self-contained applications](https://docs.microsoft.com/dotnet/core/deploying/index).
 
 Latest versions of each variant:
 
-- `microsoft/dotnet` or `microsoft/dotnet:latest` (includes SDK)
-- `microsoft/dotnet:onbuild`
-- `microsoft/dotnet:core`
-- `microsoft/dotnet:core-deps`
+* `microsoft/dotnet` or `microsoft/dotnet:latest` (includes SDK)
+* `microsoft/dotnet:sdk`
+* `microsoft/dotnet:runtime`
+* `microsoft/dotnet:runtime-deps`
 
-Here is a list of the images after a `docker pull <imagename>` on a development machine to show the various sizes. Notice, the development/build variant, `microsoft/dotnet:1.0.0-preview2-sdk` is larger as it contains the SDK to develop and build your application. The production variant, `microsoft/dotnet:core` is smaller, as it only contains the .NET Core runtime. 
-The minimal image capable of being used on Linux, `core-deps`, is quite smaller, however your application will need to copy a private copy of the .NET runtime with it. Since containers are already private isolation barriers, you will lose that optimization when running multiple dotnet based containers. 
+Here is an images list to show the various image sizes. The actual sizes will vary with each release, but SDK (`microsoft/dotnet:2.0.0-sdk`) will be larger than runtime, and runtime (`microsoft/dotnet:runtime`) will be larger than runtime-deps (`runtime-deps`).
 
-```
-REPOSITORY          TAG                     IMAGE ID            SIZE
-microsoft/dotnet    1.0.0-preview2-onbuild  19b6a6c4b1db        540.4 MB
-microsoft/dotnet    onbuild                 19b6a6c4b1db        540.4 MB
-microsoft/dotnet    1.0.0-preview2-sdk      a92c3d9ad0e7        540.4 MB
-microsoft/dotnet    core                    5224a9f2a2aa        253.2 MB
-microsoft/dotnet    1.0.0-core-deps         c981a2eebe0e        166.2 MB
-microsoft/dotnet    core-deps               c981a2eebe0e        166.2 MB
-microsoft/dotnet    latest                  03c10abbd08a        540.4 MB
-microsoft/dotnet    1.0.0-core              b8da4a1fd280        253.2 MB
+```console
+REPOSITORY          TAG                 IMAGE ID            SIZE
+microsoft/dotnet    latest              cc72b1545c62        1.64GB
```

Comment:
What is the motivation in stating the sizes of the images?  These tags are manifest backed which means the concrete image pulled differs based on the host platform (Linux/Windows, AMD64/ARM32).  This section makes no mention of this therefore the stated sizes are wrong depending on the users perspective.

Reply:
Given that, it might be less confusing if we removed this altogether.  Thoughts?

Reply:
Yes I would agree that removing the size will make it less confusing.

Reply:
Ok, as the whole point of that part was to illustrate sizes, I removed it.